### PR TITLE
ev3dev-config: Fix error when on a platform with no hardware options

### PR DESCRIPTION
**Pull request by WasabiFan**
Ref ev3dev/ev3dev#570

### CODE REVIEW

File: source/scripts/ev3dev-config.sh [hunk 1]
```diff
@@ -265,6 +265,11 @@ do_hardware_menu() {
   current_platform=$(detect_current_platform)
   get_available_hardware_options "$current_platform"
   
+  if [ ${#HARDWARE_MENU_OPTIONS[@]} -eq 0 ]; then
+    whiptail --msgbox "Sorry, there are no available hardware options for your platform." 20 60 1
```
Comment (dlech) [inline source/scripts/ev3dev-config.sh]: No need to apologize. We could just say "There are currently no hardware options for this device."

**Comment by dlech** [comment]
Tested working.

**Comment by WasabiFan** [comment]
Amended.

**Comment by dlech** [comment]
Thanks!

**Pull request merged**